dockerfiles/ubuntu/is-analytics/Dockerfile

```diff
@@ -77,4 +79,4 @@ ENV JAVA_HOME=${JAVA_HOME} \
 # expose ports
 EXPOSE 4000 9764 9444 7712 7612 11225 10006 11001 11501 8082 4041
 
-ENTRYPOINT ${WORKING_DIRECTORY}/init.sh
+ENTRYPOINT ${WSO2_SERVER_HOME}/bin/wso2server.sh
```

Comment:
@chamindi-a this should not change. The `ENTRYPOINT` should refer to `${WORKING_DIRECTORY}/init.sh`.